### ISSUE TITLE
Adding in a .gitignore file on ape initialization

### ISSUE THREAD

**Issue by franfran20** [ApeWorX/ape#917]
### Overview

Provide built in `.gitignore` file on initialization with folders and files like:
```
.build
.pytest_cache
__pycache__
.env

```

Not a big deal but some devs forget to put in their .env into a .gitignore and you don't probably always want to push your build folder all the time along with your work.

**Comment by unparalleled-js** [comment]
Yes, and it should also include everything in the default Python `.gitignore` (the one GitHub uses)

**Comment by franfran20** [comment]
Definitely

**Comment by unparalleled-js** [comment]
this got done here: https://github.com/ApeWorX/ape/pull/1175
